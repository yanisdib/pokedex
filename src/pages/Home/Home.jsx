function Home() {
    return <p>Pok√©dex</p>;
}

export default Home;