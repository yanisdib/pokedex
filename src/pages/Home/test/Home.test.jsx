import { shallow } from 'enzyme';

import Home from '../Home';


describe('Home page', () => {
    let wrapper;

    beforeEach(() => wrapper = shallow(<Home />))

    it('renders correctly without crashing', () => expect(wrapper).toMatchSnapshot())
})