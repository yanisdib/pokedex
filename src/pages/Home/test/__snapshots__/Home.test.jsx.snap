// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page renders correctly without crashing 1`] = `ShallowWrapper {}`;
